### DISCUSSION TITLE
[Support]: api connection issue

### DISCUSSION THREAD

**Discussion by sheminasalam** [blakeblackshear/frigate#13190] [General Support] [answered]
### Describe the problem you are having

I want to save the latest image in one of the frigate image entities in HA using node red. I have been trying to use the api feature for this. But it seems that I couldn't even get the basic stats from frigate server as I am not able to connect to it. I get the error from the http request node

```{"_msgid":"0d1237b904a57783","payload":"RequestError: connect ECONNREFUSED 192.168.3.20:5000 : http://192.168.3.20:5000/api/stats","topic":"","statusCode":"ECONNREFUSED"}```

I know its some authorization issue but can't get to bottom of it.

The following is the relevant log from frigate
```
2024-08-19 15:11:33.818018588  127.0.0.1 - - [19/Aug/2024:20:41:33 +0530] "" 400 0 "-" "-" "-"
2024-08-19 15:11:46.287485311  172.30.32.2 - - [19/Aug/2024:20:41:46 +0530] "GET / HTTP/1.1" 200 2651 "-" "HomeAssistantSupervisor/2024.08.0 aiohttp/3.9.5 Python/3.12" "-"
```

also I haven't given any authorization with the http request node.

### Version

Current version: 0.14.0

### Frigate config file

```yaml
na
```

### Relevant log output

```shell
na
```

### FFprobe output from your camera

```shell
na
```

### Operating system

Debian

### Install method

HassOS Addon

### Object Detector

CPU (no coral)

### Network connection

Wireless

### Camera make and model

onvif

**Comment by hawkeye217** [comment]
If you're using the HA addon, you need to enable port 5000 in the addon configuration screen.

**Reply by sheminasalam** [reply, marked as answer]
actually it worked with http://ccab4aaf-frigate:5000
